pkg/ocm/sca/sca.go

```diff
@@ -180,18 +189,26 @@ func (c *Controller) requestDataAndCheckSecret(ctx context.Context, endpoint str
 	})
 }
 
-func (c *Controller) processResponses(ctx context.Context, responses Response) error {
+func (c *Controller) processResponses(ctx context.Context, responses Response, controlPlaneArch string) error {
 	if responses.Total == 1 {
```

Comment:
This condition could be probably removed if we are fine with having the `etc-entitlement-pki` and `etc-entitlement-pki-ARCH` secrets also in the one architecture clusters.

Reply:
I would not create unnecessary resources in a cluster